TRANSLATION.md

```diff
@@ -163,9 +167,11 @@ Terminy z (?) przy sugestii są do przegadania. Jeśli wiesz, jakie powinno ich
 | roadmap | plan działania | |
 | section | podrozdział | Jeśli w ramach tego samego dokumentu, lub rozdział jeśli chodzi o inną stronę |
 | selective hydration | hydratacja selektywna | |
+| setup function | funkcja konfiguracyjna (?) | W kontekście `useEffect` |
```

Comment:
Inne propozycje:
- funkcja inicjalizacyjna
- funkcja ustawiająca
- funkcja efektu (za legacy)

Link do angielskiej wersji dokumentacji: https://react.dev/reference/react/useEffect#parameters

Reply:
Jak już to funkcja konfigurująca. I chyba przy tym bym został.